### ISSUE TITLE
Invalid yaml config upgrading from 0.4.0 to 0.4.1

### ISSUE THREAD

**Issue by parautenbach** [kucherenko/jscpd#37]
This config used to work fine with v0.4.0 until v0.4.1 was released just now:

```yaml
path:
  - .
languages:
  - javascript
  - coffeescript
exclude:
  - "node_modules/**"
  - "test/**"
  - "public/lib/**"
output: "cpd.xml"
```

```
File .../src/.cpd.yml not found in current directory, or it is broken
```

**Comment by ghusse** [comment]
Hi have the same issue.

It seems that jscpd is only looking for `.cpd.yml`, as my config file is named `.cpd.yaml` it does not find it.

If I rename it to `.cpd.yml`, the error disappears, but it does not work either. All my config is not applied at all !

Please fix this issue, as it breaks my continuous integration jobs

**Comment by parautenbach** [comment]
Thanks @ghusse for pointing out the file extension issue -- it's helpful information. I can confirm that after renaming my file, it is indeed found, but the config doesn't seem to have any effect.

**Comment by kucherenko** [comment]
sorry, I'll fix it soon

**Comment by parautenbach** [comment]
No problem -- and thank you! There is at least a working version, so there are other options in the mean time.

**Comment by kucherenko** [comment]
fixed in 0.4.2

**Comment by parautenbach** [comment]
Thank you!
